packages/amplify_datastore/android/src/test/kotlin/com/amazonaws/amplify/amplify_datastore/AmplifyDataStorePluginTest.kt

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import com.amazonaws.amplify.amplify_datastore.AmplifyDataStorePlugin
+import com.amazonaws.amplify.amplify_datastore.readMapFromFile
+import com.amazonaws.amplify.amplify_datastore.types.model.FlutterSerializedModel
+import com.amplifyframework.core.Amplify
+import com.amplifyframework.core.Consumer
+import com.amplifyframework.core.model.Model
+import com.amplifyframework.core.model.query.Page
+import com.amplifyframework.core.model.query.QueryOptions
+import com.amplifyframework.core.model.query.Where
+import com.amplifyframework.core.model.query.predicate.QueryField.field
+import com.amplifyframework.core.model.query.predicate.QueryPredicateOperation.not
+import com.amplifyframework.core.model.temporal.Temporal
+import com.amplifyframework.datastore.AWSDataStorePlugin
+import com.amplifyframework.datastore.DataStoreCategory
+import com.amplifyframework.datastore.DataStoreException
+import com.amplifyframework.datastore.appsync.SerializedModel
+import io.flutter.plugin.common.MethodChannel
+import java.lang.reflect.Field
```

Comment:
Can you add some package groupings with spaces between, to help break this up a bit?

Reply:
Done